lib/ledger_sync/adaptors/quickbooks_online/account/mapping.rb

```diff
@@ -0,0 +1,103 @@
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      module Account
+        module Mapping
+          ACCOUNT_TYPES = {
+            'bank' => 'Bank',
+            'other_current_assets' => 'Other Current Asset',
+            'fixed_asset' => 'Fixed Asset',
+            'other_asset' => 'Other Asset'
```

Comment:
Hey @jozefvaclavik  - it looks like you missed the equity, expense, liability, and revenue accounts from https://developer.intuit.com/app/developer/qbo/docs/api/accounting/most-commonly-used/account

You can see them if you scroll past the asset-related accounts